### PR TITLE
Universal/fix forward to universal

### PR DESCRIPTION
**Pull request by kaychang-unity**
---
### Purpose of this PR
Minor fix to avoid update script for UniversalRenderer to crash.
Additional fix for SSAO_DeferredRenderer.asset, which was changed to different settings by mistake a few days ago.

---
### Testing status
Loaded SRP_SmokeTest and UniversalGraphicsTest, projects update normally without errors.

---
### Comments to reviewers
I did not add a changelog as UniversalRenderer renaming has not been released to the public yet.

### CODE REVIEW

**Review by ellioman — APPROVED** [review]

**Review by hdb-unity — APPROVED** [review]
No QA testing.

Reviewed as codeowner for UniversalGraphicsTest.

The changes to UniversalGraphicsTest/Assets/Scenes/140_SSAO/Renderers/SSAO_DeferredRenderer.asset should be copied to all 5x split projects (they all contain SSAO_DeferredRenderer.asset in 140_SSAO folder)

In this case, I understand the 5x changes are not required, and will be handled in a followup PR ( 3174 ).

Ideally, the change can be reverted before landing to not affect UniversalGraphicsTest project.

**Pull request merged**
